lib/bootloader_setup.pm

```diff
@@ -35,17 +36,32 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+sub stop_grub_timeout {
```

Comment:
whatever I don't mind, but yes it's confusing, because you have to look what is stop_grub_timeout exactly doing and it's just one line of code, I don;t see any reason why to do it